src/axom/inlet/Field.hpp

```diff
@@ -314,10 +314,10 @@ class Field : public VerifiableScalar
 
   /*!
    *****************************************************************************
-   * \brief Returns whether an actual value is stored
+   * \brief Returns whether a value was provided in the input file
```

Comment:
I like this functionality, but I am not sure that it is intuitive for it to be the implementation for `operator bool()`

There is likely a global view of how this all works that I'm missing though...

Reply:
The motivation for this specific change is consistency with the logic added in #481 regarding when a `Container` is verified - I think it would be fine to move this to an explicitly named method (`inInputFile`?), but that raises the question of what `operator bool` should do, or if these objects even have an intuitive definition of `operator bool`.